data/presets/amenity/recycling/container/glass_bottles

```diff
@@ -0,0 +1,35 @@
+{
+    "icon": "maki-recycling",
+    "fields": [
+        "{amenity/recycling_container}"
+    ],
+    "moreFields": [
+        "{amenity/recycling_container}"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "bottle",
+        "bottle bank",
+        "bottle container",
+        "bottle recycling",
+        "bottle collection",
+        "glass bottle",
+        "glass bottle bank",
+        "glass bottle container",
+        "glass bottle recycling",
+        "glass bottle collection"
+    ],
+    "tags": {
+        "amenity": "recycling",
+        "recycling_type": "container",
+        "recycling:glass_bottles": "yes"
+    },
+    "reference": {
+        "key": "recycling:glass_bottles",
+        "value": "yes"
+    },
+    "name": "Single Glass Bottle Container"
```

Comment:
To keep it more in line with https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/amenity/recycling_container.json#L41 and to show that it is a subgroup of that, we could name it
```suggestion
    "name": "Recycling Container for Glass Bottles"
```

Reply:
Thanks for the hint. I updated the name accordingly.